_includes/live-demos/template/index.html

```diff
@@ -0,0 +1,3 @@
+<textarea id="template">
+  <p>Select the insert menu or toolbar icon above to insert a custom template.</p>
```

Comment:
Maybe change the initial content to:

To insert a template, either:
- From the **Insert** menu, select **Insert template...**.
- Click the **Insert template** toolbar button.